infra/build/functions/build_lib.py

```diff
@@ -327,19 +349,34 @@ def get_git_clone_step(repo_url='https://github.com/google/oss-fuzz.git',
   return clone_step
 
 
+def _armify_image_name(image_name):
```

Comment:
nit: rename to `_platform_specific_image_name(image_name, architecture)`.

Reply:
Done.